.travis.yml

```diff
@@ -6,18 +6,18 @@ env:
 matrix:
   fast_finish: true
   include:
-    - env: DM=~4.2
+    - env: DM=~6.0
       php: 5.3
-    - env: DM=~4.2
+    - env: DM=@dev
```

Comment:
Hm, ok. Simply moving the `@dev` up to PHP 5.5 should be ok. Is it ok when I do this edit without a pull request?